PhysicsTools/NanoAOD/python/taus_updatedMVAIds_cff.py

```diff
@@ -281,162 +281,25 @@
 )
 for m in patTauDiscriminationByElectronRejectionMVA62015.mapping:
     m.cut = m.cut.value().replace(antiElectronDiscrMVA6_version, antiElectronDiscrMVA6v1_version + "_gbr")
-### Put all anti-e tau-IDs into a sequence
-_patTauDiscriminationByElectronRejection2018Task = cms.Task(
-    patTauDiscriminationByElectronRejectionMVA62018Raw,
-    patTauDiscriminationByElectronRejectionMVA62018
-)
-_patTauDiscriminationByElectronRejection2015Task = cms.Task(
+
+patTauDiscriminationByElectronRejectionTask = cms.Task(
```

Comment:
Are the "2015" IDs still used anywhere?

Reply:
maybe not. we should come back to this later on though